Week9Serialization/Week9Serialization/Functional Classes/Female.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Week9Serialization
+{
+    /// <summary>
+    /// This class inherits from the Person class.
+    /// </summary>
+    [Serializable]
+    [DataContract]
+    public class Female: Person
```

Comment:
A better XML comment for this class would be something like, "Represents a female person."